linux/keyman-config/keyman_config/install_kmp.py

```diff
@@ -108,9 +109,13 @@ def _install_kmp(self, inputfile, online, language):
             raise InstallError(InstallStatus.Abort, message)
 
         extract_kmp(inputfile, self.packageDir)
-        restart_ibus()
 
-        info, system, options, keyboards, files = get_metadata(self.packageDir)
+        if is_fcitx_running():
+            restart_fcitx()
+        else:
+            restart_ibus()
+
+        info, _, _, keyboards, files = get_metadata(self.packageDir)
```

Comment:
Why the removal of the `system` and `options` parameters?

Reply:
Those are unused.